modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -0,0 +1,16 @@
+[[tuning-changelogs]]
+= Tuning Changelogs
+
+Some packages have a long list of changelog entries.
+This data is downloaded by default, but is very rarely useful information.
+In order to limit the amount of changelog metadata which is downloaded and to save disk space, you can put a limit on how many entries to keep on disk.
+
+This configuration option is in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+The parameter defaults to [systemitem]``0``, which means unlimited.
+
+----
+java.max_changelog_entries = 0
+----
+
+When you set this parameter, it will come into effect only when packages are synchronized.
+You might like to delete the cached data to remove older data.
```

Comment:
I see L3s flying with the question "how can I remove cached data" :-)

```
$> spacewalk-sql -i
SQL> delete from rhnPackageRepodata;
SQL> insert into rhnRepoRegenQueue (id, CHANNEL_LABEL, REASON, FORCE)
(select sequence_nextval('rhn_repo_regen_queue_id_seq'),
        C.label,
        'package summary modification',
        'Y'
   from rhnChannel C);
SQL> \q
$>
```

WARNING: this will regenerate all metadata and this is a very long running task. Expect several hours until it is finished. It depends on the number of channels the customer have. The task run via taskomatic in background.